src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,127 @@
+package duke;
+
+import duke.exception.*;
```

Comment:
Same as above, I think it would be better to import the specific classes that you need.

Reply:
Got it, thanks!